Getting-Started-with-Akka-http-signature.md

```diff
@@ -114,111 +114,77 @@ This will save the contents of the key in String format within the file found by
+A user can also transform his public keys to an RDF. In order to do this one must first import multiple files in order to resolve the required dependencies:
 
 The org.w3 declarations required are:
 
 ```scala
-import org.w3.banana.binder
-import org.w3.banana.binder.RecordBinder
-implicit val binder = RecordBinder
-val cert = CertPrefix[Rdf]
-```
-Several java imports from java's security and math library are also required for this process:
+import $ivy.`run.cosy::solid-client:0.1-SNAPSHOT`
+import run.cosy.auth.RSAKeys
+import com.typesafe.sslconfig.akka._
+import akka.actor._
+import akka.http.scaladsl.model.Uri
+import com.typesafe.config.ConfigFactory
+import akka.event.Logging
+import akka.stream.{ActorMaterializer, ActorMaterializerSettings,Supervision,_}
+import akka.http.scaladsl.model.{Uri=>AkkaUri,_}
 
-```scala
-import java.math.BigInteger
-import java.security.KeyFactory
-import java.security.interfaces.RSAPublicKey
-import java.security.spec.RSAPublicKeySpec
-```
+import scala.concurrent.ExecutionContext
+
+import org.w3.banana.jena.Jena
 
-After that the binder and all it's functionalities can be imported:
+import Jena._
 
-```scala
-import org.w3.banana.binder._
-import recordBinder._
+import run.cosy.solid.client.Web._
+import java.security.interfaces.RSAPublicKey
 ```
 
-Finally, after that, the user can create the Cert object that contains the required dependencies to turn a Key into a Pointed Graph
+Finally, after that, the user can create a value for the public key String collected from the .pem file in located within the .keys directory.
 
 ```scala
-@ object Cert {
-
-   implicit val rsaClassUri = classUrisFor[RSAPublicKey](cert.RSAPublicKey)
-   val factory = KeyFactory.getInstance("RSA")
-   val exponent = property[BigInteger](cert.exponent)
-   val modulus = property[Array[Byte]](cert.modulus)
-
-   implicit val binder: PGBinder[Rdf, RSAPublicKey] =
-   pgb[RSAPublicKey](modulus, exponent)(
-      (m, e) => factory.generatePublic(new RSAPublicKeySpec(new BigInteger(m), e)).asInstanceOf[RSAPublicKey],
-      key => Some((key.getModulus.toByteArray, key.getPublicExponent))
-             ) // withClasses rsaClassUri
-}
+@ val pubStr = read(home/".keys"/"pubKey.pem")
 
-defined object Cert
+pubStr: String = """MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAkZXI44b8Qh7oNXzGvmdGSan1cisdzOur
+WWWheSxvr9zHe5kyNh8UtVZeVjTsRr/SnwEwqm1KJRiu0CfnyjTZmhQZJitZDjg+sTdDx3pcxntC
+MfckgRfaG+MjODbL2VTNHzaDYHlex0VwITcPH7RjPxJZyYmAlVg+MWMfX2VOBPqdBRZxO0DyH6ka
+kENtgT0TJ9XNEVWH+gpezc66jgptz/wryzCaVobdF042TvQ5VoZC5gavUMgDuHS3TiT9LXBFT3Hg
+A18h0qaZDkwJe6mHD/aULVrZYxf9irGJQAS2aTQHO/zdRJazlMX7oWQrVRpCmcz/BLpgxz4x9IC0
+hHhTWQIDAQAB"""
 
-@ import Cert._
 ```
 
-After this all dependencies should be resolved and the user will be able to transform the keys into a Pointed Graph:
+After this all dependencies should be resolved and the user will be able to transform the keys into a Pointed Graph using the following function
```

Comment:
the function below is hardly a function worth mentioning: It's more of a piece of test code to verify that things are working correctly. That is obvious because there are no attributes passed to the funciton.